index.js

```diff
@@ -46,6 +48,10 @@ exports = module.exports = (function () {
     return getHarness().onFailure.apply(this, arguments);
   };
 
+  lazyLoad.beforeEach = function (cb) {
```

Comment:
What if there is multiple invocations on `test.beforeEach`? In the case these hooks are expected to be called in order respectively.